tests/e2e/scenarios/test_reborn_webui_v2_smoke.py

```
@@ -1249,68 +1267,190 @@ async def test_reborn_v2_ui_enter_submits_initial_and_follow_up_messages(
     await expect(assistant_messages.last).to_contain_text("I understand your request.")
 
 
-async def test_reborn_v2_automation_rename_persists_from_ui(
+async def test_reborn_v2_automation_lifecycle_persists_from_ui(
```

Ruff (0.16.1)

[warning] Too many statements (100 > 50)

(PLR0915)